.prow.sh

```diff
@@ -1,28 +1,62 @@
 #! /bin/bash
 
+CSI_PROW_SIDECAR_E2E_IMPORT_PATH="github.com/kubernetes-csi/external-provisioner/test/e2e"
+
 . release-tools/prow.sh
 
-# External-provisioner is updated to use VolumeSnapshot v1 APIs. As a result,
-# tesing on Kubernetes version < 1.20 are no longer supported because they use
-# v1beta1 or earlier VolumeSnapshot APIs.
-# TODO: The check can be removed when all Prow jobs for Kubernetes < 1.20 are removed.
-if ! (version_gt "${CSI_PROW_KUBERNETES_VERSION}" "1.19.255" || [ "${CSI_PROW_KUBERNETES_VERSION}" == "latest" ]); then
-    filtered_tests=
-    skipped_tests=
-    for test in ${CSI_PROW_TESTS}; do
-	case "$test" in
-	    parallel | serial | parallel-alpha | serial-alpha)
-		skipped_tests="$skipped_tests $test"
-		;;
-	    *)
-		filtered_tests="$filtered_tests $test"
-		;;
-	esac
+volume_mode_conversion () {
+   [ "${VOLUME_MODE_CONVERSION_TESTS}" == "true" ]
+}
+
+if volume_mode_conversion; then
+  install_snapshot_controller() {
```

Comment:
The `.prow.sh` file in every repo can override `release-tools/prow.sh`. In this case, the feature is only applicable to the provisioner. So instead of modifying `install_snapshot_controller()` for every sidecar, updating it in this file for this repo keeps the scope of the change to provisioner only. 

Installation of snapshot CRDs is taken care of in `prow.sh` over here - https://github.com/kubernetes-csi/external-provisioner/blob/master/release-tools/prow.sh#L758